.github/workflows/integration.yml

```diff
@@ -7,53 +7,80 @@ on:
   pull_request:
     branches: ["main"]
 
-env:
-    P_LIBSSH_DEBUG: /tmp/libssh-provider-debug.log
-
 jobs:
-  build:
-    name: libssh
+  libssh:
     runs-on: ubuntu-22.04
-    container: fedora:latest
+    container: docker.io/dokken/fedora-40
```

Comment:
can we rather use fedora:rawhide ?
Otherwise this will have to be constantly maintained as time apsses

Reply:
Once F40 is released we can use fedora:latest everywhere. I can try to use fedora:rawhide. I think I picked dokken/fedora-40 for some specific reason I resolved later anyway. Let me check...

Reply:
Alright, not it works fine with fedora:rawhide.